src/coreclr/src/jit/hwintrinsicarm64.cpp

```diff
@@ -379,6 +379,14 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             return gtNewScalarHWIntrinsicNode(baseType, op1, intrinsic);
         }
 
+        case NI_AdvSimd_ExtractAndNarrowLow:
```

Comment:
Why can't these be handled by the default importation logic?

Reply:
Once I added the special codegen flag it forced me to define these too.. Although I'm not quite sure what the difference is between `HW_Category_Special` and `HW_Flag_SpecialCodeGen`. Perhaps I need to have `HW_Flag_SpecialCodeGen` instead?

The confusing part I find is that both options are still partially table driven, since most of the types come from the table entries so I found the description on `HW_Flag_SpecialCodeGen` a bit vague.

Reply:
`HW_Category_Special` means it requires specialized importation logic because the default importation logic can't handle it.
`HW_Flag_SpecialCodeGen` likewise means it requires specialized codegen logic because the default codegen logic can't handle it.